src/components/card/templeStayCard/searchCardList/SearchCardListSkeleton.tsx

```
@@ -0,0 +1,18 @@
+import TempleStayCardSkeleton from '../templeStayCardSkeleton/TempleStayCardSkeleton';
+import container from './searchCardList.css';
```

ESLint

[error] `./searchCardList.css` import should occur before import of `../templeStayCardSkeleton/TempleStayCardSkeleton`

(import/order)